nemo_skills/pipeline/utils.py

```diff
@@ -255,6 +267,12 @@ def create_remote_directory(directory: str | list, cluster_config: dict):
     if isinstance(directory, str):
         directory = [directory]
 
+    # Get unmounted path of all directories
+    directory = [
+        get_unmounted_path(cluster_config, dir_path) if is_mounted_filepath(cluster_config, dir_path) else dir_path
```

Comment:
isn't the if part redundant? I think the `get_unmounted_path` already returns original if it's not mounted